docs/docs/use-static-query.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Querying Data in Components with the useStaticQuery hook
+---
+
+Gatsby v2.1.0 introduces `useStaticQuery`, a new [React Hook](https://reactjs.org/docs/hooks-intro.html) that. Just like the `StaticQuery` component, it allows your React components to retrieve data via a GraphQL query. But with the simplicity, shareability and elegance of Hooks.
```

Comment:
```suggestion
Gatsby v2.1.0 introduces `useStaticQuery`, a new Gatsby feature that provides the ability to use a [React Hook](https://reactjs.org/docs/hooks-intro.html) to query with GraphQL at _build time_. Just like the `StaticQuery` component, it allows your React components to retrieve data via a GraphQL query that will be parsed, evaluated, and the resulting data will be injected into the component. However--`useStaticQuery` uses a hook rather than a render prop!
```

Reply:
Also maybe link to StaticQuery docs when you mention it?